assets/sass/local.scss

```diff
@@ -1,7 +1,9 @@
 .govuk-main-wrapper a:not(.govuk-button, .govuk-tabs__tab):visited {
   color: $govuk-link-colour;
 }
-
+.location-bar .location-bar__location a:visited {
```

Comment:
Very minor point - just occurred to me that you could combine these two rules as they're doing the same thing; so something like:
```scss
.govuk-main-wrapper a:not(.govuk-button, .govuk-tabs__tab):visited,
.location-bar__location a:visited {
   color: $govuk-link-colour;
 }
```

Reply:
Was writing this as a suggestion, you got to it before me 😆